include/libtropic.h

```diff
@@ -304,10 +304,10 @@ lt_ret_t lt_r_mem_data_write(lt_handle_t *h, const uint16_t udata_slot, uint8_t
  * @param h           Device's handle
  * @param udata_slot  Memory's slot to be read
  * @param data        Buffer to read data into
- * @param size        Size of data to be read
+ * @param size        Number of read data bytes
```

Comment:
Maybe better wording?
- Count of valid bytes in data.
- Number of bytes read into data.